server/auth/middleware.js

```diff
@@ -141,9 +152,10 @@ const createSignInCallbackMiddleware = ({
 
 const createSignOutMiddleware = ({ logger, analyticsService }) =>
   function signOut(req, res) {
-    if (req.user?.isSignedIn()) {
+    if (req.session?.isSignedIn) {
       logger.info(`SignOutMiddleware (signOut) - User: ${req.user.prisonerId}`);
-      req.logOut();
+      req.session.isSignedIn = false;
+      // req.logOut();
```

Comment:
Commented out for now. I think we should leave users logged in though.

Reply:
happy to go with that